src/obj.cr

```diff
@@ -197,6 +197,12 @@ module ImGui
     LibImGui.Begin(name, p_open, flags)
   end
 
+  # Calls `begin`, conditionally yields to the block, then calls `end`.
+  def self.within(name : String, p_open : Bool* = Pointer(Bool).null, flags : ImGuiWindowFlags = ImGuiWindowFlags.new(0)) : Nil
```

Comment:
I am strongly considering removing the prefix from methods such as `within_menu_bar` &rarr; `menu_bar`.
There's just the problem that `within` would be renamed to nothing according to that scheme 😅
What does `begin` mean - begin a window? Should we name this `window` then? Or what do you think?